warehouse/ingest-core/src/main/java/datawave/iterators/PropogatingIterator.java

```diff
@@ -303,7 +305,10 @@ public boolean validateOptions(Map<String,String> options) {
         Preconditions.checkNotNull(env);
         Preconditions.checkNotNull(options);
         
-        shouldPropogate = !(env.getIteratorScope() == IteratorScope.majc && env.isFullMajorCompaction());
+        // Don't propagate for either scan or full major compaction. In either case, the aggregated result has combined
+        // all existing values for a key so we don't need to propagate temporary state that is only used to combine
+        // partial results with new info.
+        shouldPropogate = !(env.getIteratorScope() == IteratorScope.majc && env.isFullMajorCompaction()) && !(env.getIteratorScope() == IteratorScope.scan);
```

Comment:
I think this change makes it much more explicit which I think is important here.